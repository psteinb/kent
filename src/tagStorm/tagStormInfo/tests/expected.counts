2	1	access
2	1	age
2	1	age_units
2	1	assay
2	1	biosample_date
2	1	donor
25	25	file
25	3	format
2	2	lab
6	2	lab_kent_disassociation_protocol
6	3	lab_kent_quality
2	1	life_stage
12	7	map_ratio
8	2	organ
23	4	part
3	3	q
2	1	sex
