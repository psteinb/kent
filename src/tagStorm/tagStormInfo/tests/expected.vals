access has 2 uses with 1 vals
   2	all
age has 2 uses with 1 vals
   2	5
age_units has 2 uses with 1 vals
   2	weeks
assay has 2 uses with 1 vals
   2	RNA-seq
biosample_date has 2 uses with 1 vals
   2	2015-01-00
donor has 2 uses with 1 vals
   2	000
file has 25 uses with 25 vals
   1	ex01do00or01fa02.fq.gz
   1	ex01do00or02fa01.fq.gz
   1	ex00do00or00fa02.fq.gz
   22	(in 22 others)
format has 25 uses with 3 vals
   23	fastq
   1	tag storm
   1	bam
lab has 2 uses with 2 vals
   1	lab_2
   1	lab_1
lab_kent_disassociation_protocol has 6 uses with 2 vals
   4	fetal_liver_digest.pdf
   2	fetal_brain_digest.pdf
lab_kent_quality has 6 uses with 3 vals
   2	0.04
   2	0.01
   2	0
life_stage has 2 uses with 1 vals
   2	embryo
map_ratio has 12 uses with 7 vals
   3	0.90
   2	0.93
   2	0.92
   5	(in 4 others)
organ has 8 uses with 2 vals
   4	brain
   4	liver
part has 23 uses with 4 vals
   6	3
   6	2
   6	1
   5	(in 1 others)
q has 3 uses with 3 vals
   1	-1
   1	3
   1	1
sex has 2 uses with 1 vals
   2	male
