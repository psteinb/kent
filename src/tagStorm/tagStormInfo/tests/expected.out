stanzas	34
tags	110
storm	1274
types	15
fields	access,age,age_units,assay,biosample_date,donor,file,format,lab,lab_kent_disassociation_protocol,lab_kent_quality,life_stage,organ,part,sex
